Sources/OptionallyDecodable/OptionallyDecodable.swift

```diff
@@ -48,3 +48,5 @@ extension OptionallyDecodable: Encodable where Wrapped: Encodable {
 extension OptionallyDecodable: Equatable where Wrapped: Equatable {}
 
 extension OptionallyDecodable: Hashable where Wrapped: Hashable {}
+
+extension OptionallyDecodable: Sendable where Wrapped: Sendable {}
```

Comment:
I think this needs to be wrapped in an `#if available(swift)` so as to compile with older Swift versions that don't have `Sendable`.

Reply:
Fixed!